jobs/JGLOBAL_WAVE_POST_BNDPNTBLL

```diff
@@ -13,9 +13,15 @@ export CDATE=${PDY}${cyc}
 export MP_PULSE=0
 
 # Set COM Paths and GETGES environment
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_WAVE_PREP COM_WAVE_HISTORY COM_WAVE_STATION
-
-if [[ ! -d ${COM_WAVE_STATION} ]]; then mkdir -p "${COM_WAVE_STATION}"; fi
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+   COMIN_WAVE_PREP:COM_WAVE_PREP_TMPL \
+   COMIN_WAVE_HISTORY:COM_WAVE_HISTORY_TMPL \
+   COMOUT_WAVE_STATION:COM_WAVE_STATION_TMPL \
+   COMOUT_WAVE_HISTORY:COM_WAVE_HISTORY_TMPL
```

Comment:
I don't think we need COMOUT_WAVE_HISTORY, this could mean we can delete this from line 22 as well here.